spinaltap-mysql/src/main/java/com/airbnb/spinaltap/mysql/ColumnSerializationUtil.java

```diff
@@ -8,13 +8,17 @@
 import java.io.Serializable;
 import java.nio.ByteBuffer;
 import java.util.Map;
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.SerializationUtils;
 import org.apache.zookeeper.server.ByteBufferInputStream;
 
+/** A utility class for MySQL {@link Column} SerDe supoort. */
 @Slf4j
+@UtilityClass
 public class ColumnSerializationUtil {
```

Comment:
I wish there was an easy way to convert to using json instead, but that will cause back compatibility issues for us :(